tests/kernel/install_ltp.pm

```diff
@@ -186,6 +185,29 @@ sub install_from_git {
     assert_script_run 'cd ltp';
     assert_script_run 'make autotools';
     assert_script_run("$configure $extra_flags", timeout => 300);
+}
+
+sub install_selected_from_git {
+    prepare_ltp_git;
+    my @paths = qw(commands/insmod
+      kernel/firmware
+      kernel/device-drivers
+      kernel/syscalls/delete_module
+      kernel/syscalls/finit_module
+      kernel/syscalls/init_module);
+
+    assert_script_run('pushd testcases');
+    foreach (@paths) {
+        assert_script_run("pushd $_ && make && make install && popd", timeout => 600);
```

Comment:
nit: Instead `pushd` and `popd` I'd prefer `cd DIR` and `cd -`, but that's really personal (not asking for removing it).